tests/Valit.Tests/String/String_MinLength_Tests.cs

```diff
@@ -33,7 +33,7 @@ public void String_MinLength_Returns_Proper_Result_For_Left_Value(int value, boo
         }
 
         [Theory]
-        [InlineData(0, false)]
+        [InlineData(0, true)]
```

Comment:
A string of zero length is now considered to be at least zero characters long. The others, 4 & 8 remain false.